documentation/modules/proc-deploying-auth-services.adoc

```diff
@@ -15,10 +15,10 @@ can be `none` (allow all), `standard` (Keycloak), or `external` (not managed by
 ifeval::["{cmdcli}" == "kubectl"]
 . Create a certificate to use with the `none` authentication service. For testing purposes, you can create a self-signed certificate:
 +
-[options="nowrap"]
+[options="nowrap",subs="+quotes,attributes"]
 ----
 mkdir -p none-authservice-cert
-openssl req -new -x509 -batch -nodes -days 11000 -subj "/O=io.enmasse/CN=none-authservice.enmasse.svc.cluster.local" -out none-authservice-cert/tls.crt -keyout none-authservice-cert/tls.key
+openssl req -new -x509 -batch -nodes -days 11000 -subj "/O=io.enmasse/CN=none-authservice.{ProductNamespace}.svc.cluster.local" -out none-authservice-cert/tls.crt -keyout none-authservice-cert/tls.key
 ----
```

Comment:
<{ProductNamespace}>

Add:
"where <{ProductNamespace}> is the namespace where {ProductName} was deployed."